### ISSUE TITLE
Add addinional languages

### ISSUE THREAD

**Issue by GermanBluefox** [iobroker-community-adapters/ioBroker.snips#22]
![image](https://user-images.githubusercontent.com/4582016/61371084-3b001c80-a894-11e9-9dff-9306f72cbfbb.png)

Actually only english and german are in the list. It could be more.

**Comment by unltdnetworx** [comment]
That's true. But then I have to translate the app inside snips as well. I only speak German, English and a little bit Swedish. So I can't do it without a native speaker.

In most cases the magic is done with text2command, so this adapter also depends on the languages supported there. Please also see Issue #46 of text2command.
